### PR TITLE
Completed typescript-template-literals exercise

### PR DESCRIPTION
**Pull request by ruqaiyae**
Screenshot 2024-12-10 at 9 43 59 AM

### CODE REVIEW

File: typescript-template-literals/main.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const age: number = 25;
+
+const bio: string = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+console.log('Bio:', bio);
+
+const fruit: string = 'apple';
+const item: string = 'pen';
+
+const applePen: string = `${fruit} ${item}`;
```
Comment (shawnkost) [inline typescript-template-literals/main.ts:18]: Technically there should be no space here!

**Review by shawnkost — APPROVED** [review]
Approved, but left a comment!

File: typescript-template-literals/main.ts (hunk 1 above)
Comment (ruqaiyae) [reply typescript-template-literals/main.ts:18]: I'll make the correction.

Comment (ruqaiyae) [reply typescript-template-literals/main.ts]: Screenshot 2024-12-10 at 9 48 30 AM

**Pull request merged**